semcore/dropdown-menu/src/DropdownMenu.jsx

```diff
@@ -53,6 +53,12 @@ class DropdownMenuRoot extends Component {
         (visible) => {
           if (!visible) {
             this.ignoreTriggerKeyboardFocusUntil = Date.now() + 100;
+          } else {
+            setTimeout(() => {
+              const selectedItem = this.itemProps.find((item) => item.selected);
+              if (!selectedItem || this.asProps.highlightedIndex !== null) return;
+              this.handlers.highlightedIndex(this.itemProps.indexOf(selectedItem));
```

Comment:
findIndex maybe on 58 line? so, you'll don't need to use `indexOf` here...

Reply:
Refactored